### PR TITLE
Adds tartufo config

### PR DESCRIPTION
**Pull request by martinomburajr**
## Description

This PR adds the latest tartufo configuration.

### CODE REVIEW

**Comment by martinomburajr** [comment]
@deanboonzaier we'll need those `Tartufo` checks 💵

**Comment by deanboonzaier** [comment]
@martinomburajr done ✅ 

image

**Review by deanboonzaier — APPROVED** [review]
🥇

File: tartufo.toml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "474242e5b16f0130e7f83e21b0120f4847a7f9bc34e0d0690d63347edd0d2c2e",
+    "727ff50666905542e20439be1f27809dfda55490ff541187f6e2a5320df2215c",
+    "82d57b73f65efb39b4ed77a41c1cf3524d95e8835715c88a11e6f3fd34251a56",
+    "a28c1b4ba066b61da2c97229cac67acb35cdb8f0f4e613601a5531f44f4a4dc8",
+    "a5aad4d04a5b531056d476058776f538af0d3d5d6b0675c3e0492ec0e6966b9f",
+    "af73ed88570d89ef8cae6b2eec6935ecaa6b5607c9403295e31be2b279395835",
+    "b67e5d1cd5d01da4e170637d11d0813525bbf2fd36bc924e2fecc3c724e7678d",
+    "e2bf38db437ad8100373a6f217bab632cd7f0fa44e7f2f1157e752671c5d1a69",
+    "e673ded437d184914b5febb349d3a1ab658e8bd11b273e1bbd5fa8f1c7b8a86b",
+    "1aacf98a2d29e1fbfb40bc5f5290c96f30a571d1ab2fe5d2481fdfd0c66a29e5"
```
Comment (deanboonzaier) [inline tartufo.toml:27]: It didn't seem like there were this many findings when I browsed the source files but I guess I must have missed something 🙈

**Pull request merged**